catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -142,11 +161,11 @@ public String toString() {
   @Override
   public List<WorkspaceMetacardImpl> getWorkspaceMetacards(Set<String> workspaceIds) {
 
-    final List<Filter> filters = createFilters(workspaceIds);
+    if (!workspaceIds.isEmpty()) {
```

Comment:
May I suggest inverting the predicate and returning the empty list here early, while getting rid of the larger indented code block?